client/app/components/conference_room/event/controls/EventControls.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import EventSchema from 'proptypes/schemas/EventSchema';
+
+const InProgressControls = ({ onFinish }) => (
```

Comment:
How about splitting it to different files (I know these components are small but it is much easier to find them if they have their own file). 
Other than that - if you create a good directory hierarchy you could get rid of _somthingControls_ namespacing (`Controls` can be removed - maybe not from `EventControls` but `InProgress` or `Unstarted` I think will be self describing).